build.gradle.kts

```diff
@@ -48,14 +49,21 @@ dependencies {
     implementation("io.github.classgraph:classgraph:_")
     implementation("io.arrow-kt:arrow-core:_")
     implementation(kotlin("stdlib-jdk8"))
+
+    implementation("com.github.kittinunf.fuel:fuel:_")
+    implementation("com.github.kittinunf.fuel:fuel-coroutines:_")
+    implementation("com.github.kittinunf.fuel:fuel-gson:_")
```

Comment:
If this is the reason why we need JCenter, we may consider using khttp instead (it's on central now)

Reply:
No we can't... so if the library is not on central, I'm going to push it

Reply:
https://github.com/square/okhttp is on central, I can just use that if being on central is a pre-requisite.

Reply:
I don't want you to throw away the current implementation. That library is useful anyway on Central, so one way or another I'll get it there